js/app/interfaces/card.js

```diff
@@ -236,6 +239,24 @@ const Card = new Lang.Interface({
     },
 
     /**
+     * Method: add_style_classes_from_model
+     * Add style classes to the card depending on model properties
+     *
+     * This adds some CSS classes to cards based on model properties such as
+     * content type, that card implementations may want to use to customize the
+     * way they style themselves.
+     *
+     * Every card implementation's constructor should call this method after
+     * setting the construct properties.
+     */
+    add_style_classes_from_model: function () {
```

Comment:
Huh, maybe I got something else wrong when I tried it — I initially tried to do this with `_interface_init`. I'll take another look